base_ux/views/mail_template_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_mail_template_form" model="ir.ui.view">
```

Comment:
agregar herencia a vista tree agregando " decoration-muted="(not active)"" al tree

Reply:
Added, but also I had to add the active field to the tree view (shown as a toggle button), because the attribute forces us to do it.